src/data/markdown/translated-guides/en/02 Using k6/02 Metrics/100 Create custom metrics.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Create custom metrics
+description: How to build custom k6 metrics for each metric type.
+---
+
+Besides the [built-in metrics](/results-output/metrics-reference), you can create custom metrics.
+For example, you can create some derived metrics from the built-in metrics, or use the [Response.timings](/javascript-api/k6-http/response) object to create a metric for a specific set of endpoints.
```

Comment:
What do you mean with _derived metrics from the built-in metrics_?

I think we're missing here the 3rd case: just a custom computed metric, or customer's business-dependent metric.

Reply:
> derived metrics from the built-in metrics?

I meant something like `iterations * 2` just to cite a trivial example. I like your description better though.